### PR TITLE
Create Freetz.org.xml

### PR DESCRIPTION
**Pull request by christianheinrichs**

### CODE REVIEW

**Comment by J0WI** [comment]
Host is currently down from here.

**Comment by J0WI** [comment]
@christianheinrichs host WFM now.
Can we redirect `trac.freetz.org` to `freetz.org` or `www.freetz.org` to avoid the cert mismatch?

**Comment by christianheinrichs** [comment]
@J0WI Done.

However, visiting `freetz.org` or any of its subdomains now show a `502 Bad Gateway` error.

**Comment by J0WI** [comment]
It worked a few hours ago...

**Comment by J0WI** [comment]
It's currently up, so let's try again.

File: src/chrome/content/rules/Freetz.org.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+	Invalid certificate:
+		- svn.freetz.org
+		- trac.freetz.org
```
Comment (J0WI) [inline src/chrome/content/rules/Freetz.org.xml]: Please add `trac.freetz.org` as target host, but keep the comment.

Comment (christianheinrichs) [reply src/chrome/content/rules/Freetz.org.xml]: Done.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**